benchmark/profile_restful_api.py

```diff
@@ -154,9 +154,8 @@ def process_request(self,
         stats = []
         while not res_queue.empty():
             session_id, _stats = res_queue.get()
-            # print(f'\n{"-" * 50}\n'
-            #       f'session {session_id} stats: \n{_stats}\n{"-" * 50}\n')
-            stats.append(np.array(_stats))
+            if len(_stats) != 0:
```

Comment:
I think we need to check if `len(_stats) == 0` indicates sth is wrong with the inference engine

Reply:
We may count the total number of successful requests like https://github.com/vllm-project/vllm/blob/a41357e941b81067cd053e94da41adae470990df/benchmarks/benchmark_serving.py#L334 I can add this feature in another PR.

Reply:
The current processing method is consistent with the other scripts under https://github.com/InternLM/lmdeploy/tree/main/benchmark

Reply:
cc @lvhan028